### DISCUSSION TITLE
The status code is returning incorrectly. I expect it to return 404, but it is ultimately returning 200.

### DISCUSSION THREAD

**Discussion by LetAmericaGreatAgain** [uNetworking/uWebSockets#1810] [Q&A]
`std::string status = "404";
            res->writeStatus(std::string_view(status));
            res->end("File not found");`
The status code is returning incorrectly. I expect it to return 404, but it is ultimately returning 200.

**Comment by uNetworkingAB** [comment]
You need to call writeStatus before you call writeHeader
